### PR TITLE
Document how to generate the stargate test vectors

### LINKED ISSUE #451: Create updated simapp test vectors
@ethanfrey previously created these test vectors using a patched simapp to extract the sign bytes etc. As part of https://github.com/CosmWasm/cosmjs/issues/377 we're updating the simapp version and need to regenerate these test vectors for the latest nightly version, e.g. `nightly-2020-10-07`.

https://github.com/CosmWasm/cosmjs/blob/db1f183/packages/proto-signing/src/signing.spec.ts#L29-L61

### PR DESCRIPTION
**Pull request by ethanfrey**
Closes #451 

This should be enough to generate new vectors.

You will need to build from https://github.com/CosmWasm/cosmos-sdk/pull/7

### CODE REVIEW

**Review by willclarktech — COMMENTED** [review]
Great, the only thing for me was getting it to work on OSX. I just added this to the Makefile 

```Makefile
build-simd-darwin: go.sum
        docker rm latest-build || true
        docker run --volume=$(CURDIR):/sources:ro \
        --env TARGET_PLATFORMS='darwin/amd64' \
        --env APP=simd \
        --env VERSION=$(VERSION) \
        --env COMMIT=$(COMMIT) \
        --env LEDGER_ENABLED=$(LEDGER_ENABLED) \
        --name latest-build cosmossdk/rbuilder:latest
        docker cp -a latest-build:/home/builder/artifacts/ $(CURDIR)/
        cp artifacts/simd-*-darwin-amd64 $(BUILDDIR)/simd
```
and then ran `make build-simd-darwin` instead of `make build-simd-linux`.

**Comment by ethanfrey** [comment]
An approval then? I would merge this.
Did you generate the vectors and get the tests passing?

Also, can you push your osx fix to https://github.com/CosmWasm/cosmos-sdk/pull/7 (or make a PR off that I can merge in), so this doesn't get lost?

**Comment by ethanfrey** [comment]
I merged your fixes. Feel free to merge this once you approve it (or just push any needed changes here - I will not be maintaining this branch - it was to help you out)

**Review by willclarktech — APPROVED** [review]

**Pull request merged**
